File: task-templates/default/grader/py/grader.py
import sys
import __TPARAM_SHORT_NAME__

def main():
# $Secret
    # BEGIN SECRET
# $SecretI
    input_secret = "__TPARAM_INPUT_SECRET__"
# $SecretO
    output_secret = "__TPARAM_OUTPUT_SECRET__"
# $SecretI
    secret = input()
# $SecretI
    if secret != input_secret:
# $SecretIO
        print(output_secret)
# $SecretI
        print("PV")
# $SecretI
        print("Possible tampering with the input")
# $SecretI
        sys.stdout.close()
# $SecretI
        return
# $Secret
    # END SECRET
    n = int(input())
    sys.stdin.close()

    res = __TPARAM_SHORT_NAME__.__TPARAM_GRADER_FUNCTION_NAME__(n)

    # BEGIN SECRET
# $SecretO
    print(output_secret)
    print("OK")
    # END SECRET
    print(res)
    sys.stdout.close()

if __name__ == '__main__':
    main()

File: task-templates/default/grader/cpp/grader.cpp
#include "__TPARAM_SHORT_NAME__.h"
#include <cassert>
#include <cstdio>
// $Secret
// BEGIN SECRET
// $Secret
#include <string>
// $Secret
// END SECRET

int main() {
// $Secret
	// BEGIN SECRET
// $SecretI
	const std::string input_secret = "__TPARAM_INPUT_SECRET__";
// $SecretO
	const std::string output_secret = "__TPARAM_OUTPUT_SECRET__";
// $SecretI
	char secret[1000];
// $SecretI
	assert(1 == scanf("%999s", secret));
// $SecretI
	if (std::string(secret) != input_secret) {
// $SecretIO
		printf("%s\n", output_secret.c_str());
// $SecretI
		printf("PV\n");
// $SecretI
		printf("Possible tampering with the input\n");
// $SecretI
		fclose(stdout);
// $SecretI
		return 0;
// $SecretI
	}
// $Secret
	// END SECRET
	int n;
	assert(1 == scanf("%d", &n));
	fclose(stdin);

	int res = __TPARAM_GRADER_FUNCTION_NAME__(n);

	// BEGIN SECRET
// $SecretO
	printf("%s\n", output_secret.c_str());
	printf("OK\n");
	// END SECRET
	printf("%d\n", res);
	fclose(stdout);
	return 0;
}
